### ISSUE TITLE
Datagrid: sortable column headers can recieve focus, but cannot be activated with the keyboard. 

### ISSUE THREAD

**Issue by JBismonte** [infor-design/enterprise#4858]
**Describe the bug**
When on a datagrid with sortable columns, it can receive focus with the keyboard, but cannot be activated with the keyboard.

**To Reproduce**

1. Go to https://master-enterprise.demo.design.infor.com/components/datagrid/test-manual-sort.html
2. try to navigate to the product header column via keyboard and click enter, it doesn't trigger the sorting.

**Expected behavior**
[VPAT Recommendation]
Ensure to provide the appropriate event listeners for keyboard functionality on this focusable column header. 

**Version**

- ids-enterprise: 4.36.2

**Platform**
 - Infor Application/Team Name: Landmark Client
 - Device: (if applicable): NA
 - OS Version: Windows 10
 - Browser Name: Firefox
 - Browser Version: v85.0

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-34404

**Comment by tmcconechy** [comment]
I think this used to work as well.

**Comment by janahintal** [comment]
QA Passed. 
![image](https://user-images.githubusercontent.com/45757016/113570066-29514080-9646-11eb-83aa-a54c09b5590a.png)
![Uploading image.png…]()
